### ISSUE TITLE
'Rename or Move Image to...' doesn't preserve file extension

### ISSUE THREAD

**Issue by cryoteeth** [typora/typora-issues#5474]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css.

- [x] Searched http://support.typora.io/

----

**Describe the bug**
When attempting to rename an image, Typora will automatically replace any existing file extensions with `.png`.

**To Reproduce**
1. Right click on an embedded image that is not a `.png` file
2. Click on 'Rename or Move Image to...'
3. The file extension in the 'Save As' input has been changed to `.png`

**Expected behavior**
The file extension to remain the same.

**Desktop (please complete the following information):**
 - OS: MacOS 12.6

**Typora Version**
1.4.4

**Comment by pvginkel** [comment]
Well, I took a stab at it myself. I fixed the issue by editing the following file in notepad: `C:\Program Files\Typora\resources\appsrc\window\frame.js`.

Search for this:

```js
"",properties:["showOverwriteConfirmation"]
```

And replace it with:

```js
"",properties:["saveFile","showOverwriteConfirmation"],filters:[{name:"All Image Types",extensions:["svg","png","jpg","jpeg"]},{name:"SVG",extensions:["svg"]},{name:"PNG",extensions:["png"]},{name:"JPEG",extensions:["jpg","jpeg"]}]
```

There's a different place in the app that has a save dialog to save images. I just copied over the settings.

If you make this change yourself, and we do get a future update of this product, you'll have to make the change again. This file will be overwritten on update.
